frontend/src/lib/components/PropertyFilters/components/FilterRow.tsx

```diff
@@ -126,6 +128,23 @@ export const FilterRow = React.memo(function FilterRow({
                                         >
                                             {`${item['alias']}::${item['regex']}`}
                                         </FilterButton>
+                                    ) : useLemonButton ? (
+                                        <LemonButton
+                                            ref={setRef}
+                                            onClick={() => setOpen(!open)}
+                                            className="new-prop-filter"
+                                            data-attr={'new-prop-filter-' + pageKey}
+                                            type="secondary"
+                                            icon={
+                                                <IconPlus
+                                                    color="var(--primary)"
+                                                    style={{ marginRight: '-0.25rem' }}
+                                                    fontSize={14}
+                                                />
```

Comment:
It's cleaner to use only CSS props for styling instead of some HTML (`color` and `fontSize` here) and some CSS.
But regarding font size and margin, it seems in the design it's the default – let's not override them for individual buttons unless completely necessary, as then various inconsistencies creep in.